README.md

```diff
@@ -59,7 +59,7 @@ The data model/DB schema lives in `j-db-client/prisma/schema`, server side code
 
 2. Update your `.env` file with your new postgres username & password. Copy and paste your root `.env` into `j-db-client/prisma/.env`. You should end up with two of the same file, one in root and the other in `j-db-client`.
 
-3. Finally, from the `j-db-client` directory, apply database migrations to your new database instance:
+3. Finally, from the root of the project, apply database migrations to your new database instance:
```

Comment:
I verified that this does in fact work from the root.

Reply:
Great!